Include/pyerrors.h

```diff
@@ -108,7 +108,9 @@ PyAPI_DATA(PyObject *) PyExc_NotImplementedError;
 PyAPI_DATA(PyObject *) PyExc_SyntaxError;
 PyAPI_DATA(PyObject *) PyExc_IndentationError;
 PyAPI_DATA(PyObject *) PyExc_TabError;
+#if !defined(Py_LIMITED_API)
 PyAPI_DATA(PyObject *) PyExc_IncompleteInputError;
```

Comment:
Does it need to be exported at all? An extension can always do `PyObject_GetAttrString(Py_GetBuiltins(), "_IncompleteInputError"))` (or whatever the "GetBuiltins" call actually is) if it needs a reference.

Removing from the limited API is definitely the right thing. Luckily we caught it pre-release.

Reply:
We are using it in the parser so it needs to be exported in the headers somehow. Or do you mean we can just use "extern" for it?

Reply:
Yeah, just `extern` would mean it's available in builds that are also building its source file, while `PyAPI_` means it's exported for use by other apps that link to our binaries. (I think the distinction is more strictly enforced on Windows than POSIX, but we can at least capture our intent.)